src/VisualStudio/Core/Def/Implementation/Library/ObjectBrowser/AbstractListItemFactory.cs

```diff
@@ -661,8 +661,9 @@ public ImmutableArray<ObjectListItem> GetReferenceListItems(ObjectListItem paren
                 return ImmutableArray<ObjectListItem>.Empty;
             }
 
-            var builder = ImmutableArray.CreateBuilder<ObjectListItem>();
+            var builder = ArrayBuilder<ObjectListItem>.GetInstance();
 
+            // Don't show previous script compilations in the references list.
```

Comment:
Well, it makes it really conspicuous this code here thinks about it when it otherwise wouldn't have to at all.